fastlane_core/lib/fastlane_core/device_manager.rb

```diff
@@ -180,6 +180,12 @@ def reset
         `xcrun simctl erase #{self.udid}`
         return
       end
+
+      def delete
+        UI.message("Deleting #{self}")
+        `xcrun simctl delete #{self.udid}`
```

Comment:
Sorry for the back and forth here @casz! But do we still need to make sure it is shut down? In my last comment I just meant that I dont think we needed the check for if it's booted. Sorry if I wasnt clear there. Does that make sense? Have you tested this while simulators are running?

Reply:
I tried with simulators on, and they do get killed, but Yes, again it makes sense to actually still call shutdown properly. I simply misunderstood you 👍

Reply:
`An error was encountered processing the command (domain=com.apple.CoreSimulator.SimError, code=164):
Unable to shutdown device in current state: Shutdown`

When a machine is already shutdown

Reply:
A nice shutdown is better, if you just delete it. and have xcode running it gets unhappy with error 👍